src/main/java/md/maib/entity/Customer.java

```diff
@@ -0,0 +1,130 @@
+    private Long id;
+
+    @Column(name = "cvv", nullable = false)
+    private String cvv;
+
+    @Column(name = "first_name", nullable = false)
+    private String firstName;
+
+    @Column(name = "last_name", nullable = false)
+    private String lastName;
+
+    @Column(name = "pan", nullable = false, unique = true, length = 16)
+    private String pan;
+
+    @OneToMany(mappedBy = "customer", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+    private List<Transaction> transactions;
+
+    public static class Builder {
+        private Long id;
+        private String firstName;
+        private String lastName;
+        private String pan;
+        private String cvv;
+
+        public Builder id(Long id) {
+            this.id = id;
+            return this;
+        }
+
+        public Builder firstName(String firstName) {
+            this.firstName = firstName;
+            return this;
+        }
+
+        public Builder lastName(String lastName) {
+            this.lastName = lastName;
+            return this;
+        }
+
+        public Builder pan(String pan) {
+            this.pan = pan;
+            return this;
+        }
+
+        public Builder cvv(String cvv) {
+            this.cvv = cvv;
+            return this;
+        }
+
+        public Customer build() {
+            Customer customer = new Customer();
+            customer.setId(this.id);
+            customer.setFirstName(this.firstName);
+            customer.setLastName(this.lastName);
+            customer.setPan(this.pan);
+            customer.setCvv(this.cvv);
+            return customer;
+        }
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public String getPan() {
+        return pan;
+    }
+
+    public void setPan(String pan) {
+        this.pan = pan;
+    }
+
+    public String getCvv() {
+        return cvv;
+    }
+
+    public void setCvv(String cvv) {
+        this.cvv = cvv;
+    }
+
+    public String getFullName() {
+        return firstName + " " + lastName;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Customer customer = (Customer) o;
+        return Objects.equals(id, customer.id) && Objects.equals(cvv, customer.cvv) && Objects.equals(firstName, customer.firstName) && Objects.equals(lastName, customer.lastName) && Objects.equals(pan, customer.pan) ;
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id, cvv, firstName, lastName, pan, transactions);
```

Comment:
Transactions - is growing list, making hash gonna grow to be more more and complicate operation, so I not sure that it gonna be used here. @taragrammatonn ?